### ISSUE TITLE
On all the dialogues make there more space between the Mixtec column and the English column

### ISSUE THREAD

**Issue by ejohnmartin** [mixtecnerds/health_in_mixtec#17]
Like... make the columns wider, too.  Wider and make more space between them.

**Comment by ejohnmartin** [comment]
No, like, there is the Mixtec column and the English column.  They are too close together.  Make white space between them.

**Comment by amundo** [comment]
sorry, i see what you mean now. should be fixed.

**Comment by ejohnmartin** [comment]
MOAR SPACE!!!  MOOOOAAAARR!!!!

**Comment by amundo** [comment]
<img width="1051" alt="screen shot 2017-01-27 at 12 41 07 pm" src="https://cloud.githubusercontent.com/assets/30924/22381062/063ae83a-e48e-11e6-9df7-8bb125e8a8ca.png">

Is this layout better?

**Comment by amundo** [comment]
beep beep
